utils/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "@nodejs/utils",
+  "version": "0.0.0",
+  "description": "Some utilities for Node.js userland migrations.",
+  "exports": {
+    "./logger": {
+      "import": "./src/logger.ts"
+    },
+    "./spawn-promisified": {
+      "import": "./src/spawn-promisified.ts"
+    },
+    "./snapshots": {
+      "import": "./src/snapshots.ts"
+    }
+  },
```

You **should** be able to not pass `exports` and have it work fine.

If you have to use exports, this should work:
```suggestion
  "exports": {
    "./*": ["./*.ts"]
  },
```